src/Lykke.Logs/Loggers/LykkeSlack/SlackLogger.cs

```diff
@@ -87,10 +87,8 @@ private static string BuildMessage(LogEntryParameters parameters, Exception exce
             if (!string.IsNullOrWhiteSpace(parameters.Context))
             {
                 if (sb.Length > 0)
-                {
                     sb.AppendLine();
-                }
-
+                sb.Append("Context: ");
```

Comment:
This info seems excess for me from the visual point and from the point of Slack message length constraint